### PR TITLE
fix(pipelined): Updating pipelined production configuration

### PR DESCRIPTION
**Pull request by mehul-jindal**
Signed-off-by: mehul-jindal-wavelabs <mehul.jindal@wavelabs.ai>

## Summary

Added missing pipelined configuration in production yml file.

### CODE REVIEW

File: cwf/gateway/docker/python/Dockerfile [hunk 1]
```diff
@@ -111,12 +111,9 @@ RUN python3.8 -m pip install --no-cache-dir \
     click \
     pycares \
     python-dateutil \
-    aioeventlet>=0.4 \
+    aioeventlet@git+https://github.com/magma/deb-python-aioeventlet@86130360db113430370ed6c64d42aee3b47cd619 \
```
Comment (sebathomas) [inline cwf/gateway/docker/python/Dockerfile]: This was also fixed on master. Might be a good idea to rebase.

Comment (mehul-jindal) [reply cwf/gateway/docker/python/Dockerfile]: Sorry about that. It's been rebased now.

File: lte/gateway/configs/pipelined.yml_prod [hunk 2]
```diff
@@ -167,12 +171,20 @@ he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 
 ovs_gtp_stats_polling_interval: 180
 
+# Make as a True when gnb ip will support for uplink
+ovs_multi_tunnel: False
```
Comment (panyogesh) [inline lte/gateway/configs/pipelined.yml_prod:175]: Can you please check if without  'ovs_multi_tunnel' things are working. Is it a must needed.

File: lte/gateway/configs/pipelined.yml_prod [hunk 3]
```diff
@@ -167,12 +171,20 @@ he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
...
+# Paging timeout value
+paging_timeout: 30
+classifier_controller_id: 5
```
Comment (panyogesh) [inline lte/gateway/configs/pipelined.yml_prod:179]: Put some comments for more information about the variables.

Comment (panyogesh) [reply lte/gateway/configs/pipelined.yml_prod:179]: Also lets make sure these parameters are explained in the Documents.

File: lte/gateway/configs/pipelined.yml_prod (hunk 2 above)
Comment (mehul-jindal) [reply lte/gateway/configs/pipelined.yml_prod:175]: This flag is used whenever uplink flow is added in ovs.

File: lte/gateway/configs/pipelined.yml_prod (hunk 3 above)
Comment (mehul-jindal) [reply lte/gateway/configs/pipelined.yml_prod:179]: Sure, we'll explain these parameters in documents.

**Review by koolzz — APPROVED** [review]
lgtm

**Pull request merged**